content/docs/command-reference/metrics/diff.md

```diff
@@ -21,23 +21,18 @@ positional arguments:
 ## Description
 
 This command provides a quick way to compare metrics among experiments in the
-repository history. It requires that Git is being used to version the metrics.
+repository history. All metrics defined in `dvc.yaml` are used by default. The
+comparison shown by this command includes the old and new value, and the
```

Comment:
are you sure we include old?

Reply:
No. This is wrong actually, thanks. Fixing:
```suggestion
comparison shown by this command includes the new value, and the
```